src/config/dynamic-theme-fixes.config

```diff
@@ -287,6 +287,65 @@ IGNORE INLINE STYLE
 
 ================================
 
+*.index-education.net
+
+INVERT
+.conteneurAbs>canvas
+
+CSS
+body:not(.ThemeAccessible) .ThemeCat-edt,
+body:not(.ThemeAccessible) .ThemeCat-edt .disable-dark-mode,
+body:not(.ThemeAccessible) .ThemeTurquoise,
+body:not(.ThemeAccessible) .ThemeTurquoise .disable-dark-mode {
+    --darkreader-text--theme-claire-scalePlus80: ${#e8e6e3} !important;
+}
+body:not(.ThemeAccessible) .ThemeCat-resultat,
+body:not(.ThemeAccessible) .ThemeCat-resultat .disable-dark-mode,
+body:not(.ThemeAccessible) .ThemeJaune,
+body:not(.ThemeAccessible) .ThemeJaune .disable-dark-mode {
+    --darkreader-text--theme-claire-scalePlus80: ${#cdc8c2} !important;
+}
+body:not(.ThemeAccessible) .ThemeProduitEDT,
+body:not(.ThemeAccessible) .ThemeProduitEDT .disable-dark-mode,
+body:not(.ThemeAccessible) .ThemeEDT,
+body:not(.ThemeAccessible) .ThemeEDT .disable-dark-mode,
+body:not(.ThemeAccessible) .ThemeCat-viescolaire,
+body:not(.ThemeAccessible) .ThemeCat-viescolaire .disable-dark-mode,
+body:not(.ThemeAccessible) .ThemeBleu,
+body:not(.ThemeAccessible) .ThemeBleu .disable-dark-mode {
+    --darkreader-text--theme-claire-scalePlus80: ${#ccc7c1} !important;
+}
+body:not(.ThemeAccessible) .ThemeCat-pedagogie,
+body:not(.ThemeAccessible) .ThemeCat-pedagogie .disable-dark-mode,
+body:not(.ThemeAccessible) .ThemeViolet,
+body:not(.ThemeAccessible) .ThemeViolet .disable-dark-mode {
+    --darkreader-text--theme-claire-scalePlus80: ${#cecac3} !important;
+}
+.ObjetBandeauPied .knowledge-container .ibp-pill {
+    border-color: ${#a49d91} !important;
+}
+.ObjetBandeauPied hr {
+    background-color: ${#2f323470} !important;
+}
+:root,
+:root body.dark-mode .disable-dark-mode {
+    --theme-neutre-claire: ${#c6caca} !important;
+    --theme-neutre-moyen2: ${#b4b4b4} !important;
+}
+.jfk-bubble.gtx-bubble,
+.captcheck_answer_label > input + img,
+span#closed_text > img[src^="https://www.gstatic.com/images/branding/googlelogo"],
+span[data-href^="https://www.hcaptcha.com/"] > #icon,
+img.Wirisformula,
+.ibe_gauche>* {
+    filter: invert(94%) hue-rotate(180deg) contrast(90%) !important;
```

Comment:
Why not include it in INVERT section which won't be problematic when light scheme is applied in Dark Reader?
Standard design:
![obraz](https://github.com/user-attachments/assets/625cf6ce-acdb-4665-a9d3-71f65f00b26a)

New Design:
![obraz](https://github.com/user-attachments/assets/ae1fd1b4-4fdc-4d96-9745-86532b92f515)

Reply:
Like I wrote it stays dark in light scheme:
![obraz](https://github.com/user-attachments/assets/774031e3-8aad-4153-87b8-fc00be4be6f9)

And when this selector is moved to INVERT section:
![obraz](https://github.com/user-attachments/assets/02a1c829-3aa4-4b0b-a808-e36a86ba6404)

So move it to INVERT please.